src/itemview_formaction.cpp

```diff
@@ -211,7 +211,8 @@ void itemview_formaction::process_operation(operation op, bool automatic, std::v
 	case OP_OPENINBROWSER:
 		LOG(level::INFO, "view::run_itemview: starting browser");
 		v->set_status(_("Starting browser..."));
-		v->open_in_browser(item->link());
+		if (!v->open_in_browser(item->link()))
+			v->show_error(_("Browser failed to open the link!"));
 		v->set_status("");
```

Comment:
In case of error, this call to `show_status` will overwrite the error message. This also applies to a few edits below.